### ISSUE TITLE
More info please!

### ISSUE THREAD

**Issue by backcountrymountains** [LouDnl/Tuya-OEM-PTZ-IPCamera#1]
Did you open it up and connect via serial? Does it take special haxor tools? Help?

Thanks!

**Comment by LouDnl** [comment]
Yes I opened it up, no it is no longer working, I used no hacker tools, just an spi flash programmer and clips to lock onto the flash chip.

**Comment by LouDnl** [comment]
![20220730_162718.jpg](https://github.com/user-attachments/assets/6998e9f7-2843-4236-b81b-515885324c28)
